### DISCUSSION TITLE
What do I need to change to have the bot name the channel as the truncated original message?

### DISCUSSION THREAD

**Discussion by jasikpark** [MarcusOtter/discord-needle#17] [Q&A] [answered]
i guess https://github.com/MarcusOtter/discord-auto-threader/blob/main/src/index.ts?L45 would do it

**Comment by MarcusOtter** [comment, marked as answer]
This could be achieved by setting the `name` of the thread to a truncated `message.content`

[Thread name](https://github.com/MarcusOtter/discord-auto-threader/blob/main/src/index.ts#L49)
[Message content](https://github.com/MarcusOtter/discord-auto-threader/blob/main/src/index.ts#L55)

EDIT: The title command has now been implemented with https://github.com/MarcusOtter/discord-needle/pull/12, so you can use that to set the title of a thread manually.

**Reply by MarcusOtter** [reply]
(Also, I'm planning on making this fully customizable with the config)

**Comment by jasikpark** [comment]
Ok, thanks! You know, I suppose it'd be more useful to setup the `/title` command to change the thread title before anything else!
